src/script_opt/CPP/Exprs.h

```diff
@@ -142,6 +142,13 @@ std::unordered_map<const EnumType*, std::unordered_map<int, int>> enum_val_mappi
 // outer map).
 int num_ev_mappings = 0;
 
+// Information captured for generating entries in "enum_mapping".
+struct EnumMappingInfo {
+    int enum_type; // as a global offset
+    std::string enum_name;
+    bool create_if_missing;
+};
+
 // For each entry in "enum_mapping", the EnumType (as a global offset) and
 // name associated with the mapping.
-std::vector<std::pair<int, std::string>> enum_names;
+std::vector<EnumMappingInfo> enum_names;
```

Comment:
Total flyby, but shouldn't this have an `extern` in case Exprs.h is included in multiple .cc files? There's no `#pragma once`, so not sure, maybe this is on purpsoe.

Reply:
Those headers are all included in only one place, as part of the `CPPCompile` class (via `Compile.h`). I split them out a while back when a reviewer pointed out it would be helpful if the different source files for various `CPPCompile` methods had corresponding headers focused on those methods (and associated member variables).